README.md

```diff
@@ -32,41 +29,57 @@ hey!
 
 ### Usage
 ```
-$ iptb --help
-
 NAME:
-	iptb - The IPFS TestBed
+   iptb - iptb is a tool for managing test clusters of ipfs nodes
 
 USAGE:
-	iptb [global options] command [command options] [arguments...]
+   iptb [global options] command [command options] [arguments...]
+
+VERSION:
+   0.0.0
 
 COMMANDS:
-	init			create and initialize testbed configuration
-	start			start up all testbed nodes
-	kill, stop		kill a specific node (or all nodes, if none specified)
-	restart			kill all nodes, then restart
-	shell			spawn a subshell with certain IPFS environment variables set
-	get			get an attribute of the given node
-	connect			connect two nodes together
-	dump-stack		get a stack dump from the given daemon
-	help, h			show a list of subcommands, or help for a specific subcommand
+     fast     create default bench and initialize
```

Comment:
Right now there is `iptb bench create [other flags] --init`. The `fast` command was added (name mostly a placeholder, though `auto` might be a good choice) to bring back the functionality of `iptb init`, as currently it maps directly to the node interface method `Init`. It would be nice to keep that exposed as a directly way to invoke that method without overloading it too much. This could also be fixed by changing out the `Core#Init` method name something else, and keeping `init` as the quick and easy way to get started with iptb.

Basically `iptb fast` is almost a direct mapping to `iptb bench create --init`, it just doesn't accept the idea of additional configuration through `--attr` flags.